syntax = "proto3";

message FileLink {
    string project_id = 1;
    string file_id = 2;
};

message SignedURL {
  string url = 1;
};

service CDN {
  rpc GetFileLink (FileLink) returns (SignedURL);
};